en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -101,11 +101,19 @@ This section involves setting up the Key Manager node and enabling it to work wi
 2.  If you wish to encrypt the Auth Keys (access tokens, client secrets, and authorization codes), see [Encrypting OAuth Keys]({{base_path}}/learn/api-security/oauth2/encrypting-oauth2-tokens/).
 
 
-3.  If you need to configure High Availability (HA) for the Key Manager, use a copy of the active instance configured above as the second Key Manager active instance and configure a load balancer fronting the two Key Manager instances.
+3. Optionally Add following configuration to enable Distributed JMS based cache invalidation within Key Manager nodes.
```

Comment:
Do we need to say 'JMS based' cache invalidation? because if that is the only option do we need to explicitly mention that it is JMS based?